src/libraries/System.Net.Security/tests/FunctionalTests/TransportContextTest.cs

```diff
@@ -50,6 +49,10 @@ private static void CheckTransportContext(TransportContext context)
             {
                 Assert.True(cbt2 == null, "ChannelBindingKind.Unique token data is not expected on OSX platform.");
             }
+            else if (OperatingSystem.IsAndroid())
```

Comment:
I would probably merge it with the OSX branch above and remove platform from the message or use interpolated string. We may get more in the future.